src/TConsole.cpp

```diff
@@ -122,21 +120,27 @@ TConsole::TConsole(Host* pH, bool isDebugConsole, QWidget* parent)
         mStandardFormat.flags &= ~(TCHAR_STRIKEOUT);
     } else {
         setWindowTitle(tr("Non Debug Console"));
-        if (parent) {
-            mIsSubConsole = true;
+        if (mType & (ErrorConsole|SubConsole|UserWindow)) {
```

Comment:
Not sure that the ErrorConsole type should be here as I do not think it should be added to the "main" console's `mSubConsoleList` though give that that is not used anywhere it is probably due for removal anyhow.

Reply:
I don't think it matters then - we can leave it be